src/ol_infrastructure/applications/xpro/Pulumi.applications.xpro.Production.yaml

```diff
@@ -13,6 +13,7 @@ config:
     DRIVE_WEBHOOK_RENEWAL_PERIOD_MINUTES: 720
     FEATURE_ENABLE_ENTERPRISE: "False"
     FEATURE_ENABLE_EXTERNAL_COURSE_SYNC: "False"
+    FEATURE_ENROLLMENT_WELCOME_EMAIL: "False"
```

Comment:
This should ideally disable the enrollment welcome email [feature](https://github.com/mitodl/mitxpro/pull/3017) from Prod until it is tested on QA.